src/callbacks/callbacks.jl

```diff
@@ -524,3 +526,16 @@ function gc_func(integrator)
     )
     return nothing
 end
+
+
+function update_surface_temp!(integrator)
+    d = 10 # ocean mixed layer depth [m]
+    ρₒ = 1020 # ocean density [kg / m³]
+    cₚₒ = 4184 # ocean heat capacity [J/(kg * C°)]
+
+    (; ᶠradiation_flux, Δt, sfc_setup, net_energy_flux_sfc) = integrator.p
```

Comment:
```suggestion
    (; ᶠradiation_flux, Δt, sfc_setup) = integrator.p
```

Reply:
Thanks for the comments, I just pushed to fix the documentation!